package.json

```diff
@@ -11,15 +11,24 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
+    "awesome-typescript-loader": "^5.2.1",
```

Comment:
Babel7 is now supporting TS, no need awesome-typescript-loader. 
Add "@babel/typescript" to .babelrc